modules/UI/videolayout/RemoteVideo.js

```diff
@@ -109,7 +108,7 @@ RemoteVideo.prototype._generatePopupContent = function () {
     // Delegate event to the document.
     $(document).on("click", "#mutelink_" + this.id, function(){
 
-        if (this.isMuted)
+        if (this.isAudioMuted)
             return;
```

Comment:
Was this a bug, or this is how it is suppose to work?
It was using this.isMuted, which is handled in RemoteVideo by this.isVideoMuted, but here we use this.isAudioMuted?